pkg/apis/v1beta1/labels.go

```diff
@@ -46,12 +46,23 @@ const (
 	NodePoolHashAnnotationKey          = Group + "/nodepool-hash"
 )
 
+// Karpenter specific taints
```

Comment:
musing: But this is kind-of ceasing to become just a "labels" file. Should we separate taints into "taints.go" or consider renaming this from "labels.go"